app/steps/ui/startpage/template.html

```diff
@@ -17,7 +17,7 @@ <h1 class="form-title heading-large">{{content.header}}</h1>
     <p>{{content.paragraph2 | safe}}</p>
 
     <ul class="panel panel-border-wide">
-        {{content.otherServices | safe}}
+        <li>{{content.otherServices | safe}}</li>
     </ul>
```

Comment:
As there is only one list item here, do we need to use a list? Perhaps we'd be better to use a generic `<div>` tag as is done in the GOV.UK elements. See http://govuk-elements.herokuapp.com/typography/ under 'Inset text' for an example.

Reply:
Yes, we do in this instance.  This is due to the content containing two hyperlinks.  If it only contained one then it could be a div or such.  I've changed it back so it now passes Pa11y.